_data/foot.yml

```diff
@@ -0,0 +1,13 @@
+tail:
+    - title: Install
+      url: http://jupyter.readthedocs.org/en/latest/install.html
+    - About
+    - Connect
+    - title: Documentation
+      url: http://jupyter.readthedocs.org/en/latest/index.html
+    - title: NBViewer
+      url: https://nbviewer.jupyter.org
+    - Widgets
+    - title: Blog
+      url: https://blog.jupyter.org
+    - Donate
```

Comment:
If you can ll it tail, there is no reason to move it to it's own file. That might be easier to keep in in nav.yml, no ? 
Up to you.